osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/SecretManagementService.java

```diff
@@ -38,144 +47,230 @@
 public class SecretManagementService implements SecurityKeyService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SecretManagementService.class);
-    private final RsaEncryptionProvider rsaEncryptionProvider;
+    private final RsaEncrypter soapRsaEncrypter;
     private final SecretManagementClient secretManagementClient;
 
-    public SecretManagementService(RsaEncryptionProvider rsaEncryptionProvider,
-            SecretManagementClient secretManagementClient) {
-        this.rsaEncryptionProvider = rsaEncryptionProvider;
+
+    //@Autowired
+    //private EncryptionService aesEncryptionService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)